src/main/java/org/prebid/server/spring/config/SettingsConfiguration.java

```diff
@@ -242,9 +243,9 @@ SettingsCache ampSettingsCache(ApplicationSettingsCacheProperties cachePropertie
     @Configuration
     @ConditionalOnProperty(prefix = "settings.in-memory-cache", name = "notification-endpoints-enabled",
```

Comment:
Propose to modify config calss like:
```
@Configuration
    @ConditionalOnProperty(prefix = "admin", name = "port")
    public static class AdminServerConfiguration {

        private static final Logger logger = LoggerFactory.getLogger(AdminServerConfiguration.class);

        @Autowired
        private ContextRunner contextRunner;

        @Autowired
        private Vertx vertx;

        @Autowired
        private BodyHandler bodyHandler;

        @Autowired
        private VersionHandler versionHandler;

        @Autowired(required = false)
        private SettingsCacheNotificationHandler cacheNotificationHandler;

        @Autowired(required = false)
        private SettingsCacheNotificationHandler ampCacheNotificationHandler;

        @Value("${admin.port}")
        private int adminPort;

        @Bean
        @ConditionalOnProperty(prefix = "settings.in-memory-cache", name = "notification-endpoints-enabled",
                havingValue = "true")
        SettingsCacheNotificationHandler cacheNotificationHandler(SettingsCache settingsCache) {
            return new SettingsCacheNotificationHandler(settingsCache);
        }

        @Bean
        @ConditionalOnProperty(prefix = "settings.in-memory-cache", name = "notification-endpoints-enabled",
                havingValue = "true")
        SettingsCacheNotificationHandler ampCacheNotificationHandler(SettingsCache ampSettingsCache) {
            return new SettingsCacheNotificationHandler(ampSettingsCache);
        }

        @Bean
        VersionHandler versionHandler() {
            return VersionHandler.create("git-revision.json");
        }

        @PostConstruct
        public void startAdminServer() {
            logger.info("Starting Admin Server to serve requests on port {0,number,#}", adminPort);

            final Router router = Router.router(vertx);
            router.route().handler(bodyHandler);
            router.route("/version").handler(versionHandler);
            if (cacheNotificationHandler != null) {
                router.route("/storedrequests/openrtb2").handler(cacheNotificationHandler);
            }
            if (ampCacheNotificationHandler != null) {
                router.route("/storedrequests/amp").handler(ampCacheNotificationHandler);
            }

            contextRunner.<HttpServer>runOnServiceContext(future ->
                    vertx.createHttpServer().requestHandler(router::accept).listen(adminPort, future));

            logger.info("Successfully started Admin Server");
        }
    }
```
So, we separate cache notification from version endpoint.